libs/ledger-live-common/src/families/stacks/bridge/utils/misc.ts

```diff
@@ -131,18 +133,22 @@ export const mapTxToOps =
   };
 
 export const getAccountShape: GetAccountShape = async info => {
-  const { initialAccount, address, currency, rest = {}, derivationMode } = info;
-
-  const publicKey = reconciliatePublicKey(rest.publicKey, initialAccount);
+  const { initialAccount, currency, rest = {}, derivationMode } = info;
+  // for bridge tests specifically the `rest` object is empty and therefore the publicKey is undefined
+  // reconciliatePublicKey tries to get pubKey from rest object and then from accountId
+  const pubKey = reconciliatePublicKey(rest.publicKey, initialAccount);
+  invariant(pubKey, "publicKey is required");
 
   const accountId = encodeAccountId({
     type: "js",
     version: "2",
     currencyId: currency.id,
-    xpubOrAddress: publicKey,
+    xpubOrAddress: pubKey,
     derivationMode,
   });
 
+  const address = getAddressFromPublicKey(pubKey);
```

Comment:
So this is the real diff here, address comes from the pubkey instead of AccountShapeInfo

Reply:
Sorry for the late reply @hedi-edelbloute!! You understand correct, the above is the core change. We in past always assumed that the xpub/Address will always be available but it seems in cases where accounts are restored using accountId only one can be found. This eliminates the need for both to be present and will now get the address from pubkey which is stored in the accountId string.